reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -424,7 +424,7 @@
 </script>
     <div id="report-container">
       <div class="ort-report-label">Scan Report</div>
-      <div>Created by <strong>ORT</strong>, the <a href="http://oss-review-toolkit.org/">OSS Review Toolkit</a>, version <REPLACE_ORT_VERSION> on <REPLACE_TIMESTAMP>.</div>
+      <div>Created by <strong>ORT</strong>, the <a href="http://oss-review-toolkit.org/">OSS Review Toolkit</a>, version 0.1.0-SNAPSHOT on <REPLACE_TIMESTAMP>.</div>
```

Comment:
This needs to stay `<REPLACE_ORT_VERSION>`.

Reply:
Done.